### DISCUSSION TITLE
I'm getting an error when starting the Mort application

### DISCUSSION THREAD

**Discussion by Chikrein** [killkimno/MORT#56] [Q&A]
![image](https://github.com/user-attachments/assets/a298ef0e-b774-4468-89bd-b461c39bcd21)
![image](https://github.com/user-attachments/assets/b36bafc0-1d9a-4017-9c29-1823c8537a4b)

**Comment by kmonkeyhead** [comment]
you need install packages

https://aka.ms/vc14/vc_redist.x64.exe
https://aka.ms/vc14/vc_redist.x64.exe
 https://aka.ms/vs/16/release/vc_redist.x86.exe
